server/game/cards/02.6-MotE/AlongTheRiverOfGold.js

```diff
@@ -3,12 +3,11 @@ const ProvinceCard = require('../../provincecard.js');
 class AlongTheRiverOfGold extends ProvinceCard {
     setupCardAbilities() {
         this.action({
-            title: 'During water Province chose a participating caracter, switch character base skils until turn',
-            condition: () => this.game.currentConflict.conflictProvince.getElement() === 'water',
-            target:{
+            title: 'During water Province chose a participating caracter, switch character base skils until end of turn',
+            condition: () => this.game.currentConflict.conflictProvince.getElement() === 'water' && this.game.currentConflict(),
+            target: {
                 cardType: 'character',
-                cardCondition: card => card.location === 'play area' && card.cardData.military !== undefined && card.cardData.political !== undefined
-
+                cardCondition: card => card.isParticipating && card.cardData.military !== undefined && card.cardData.political !== undefined
```

Comment:
isParticipating is function, so it needs to have the **()** at the end: `card.isParticipating()`